docs/src/components/InteractivePlayground/useClampPlayground/Example.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { Button, StyleSheet, View, Text } from 'react-native';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withClamp,
+  withSpring,
+  cancelAnimation,
+  withRepeat,
+  withSequence,
+  withTiming,
+  withDelay,
+} from 'react-native-reanimated';
+
+const BOX_START = 0;
+const BOX_STOP = 200;
+const BOX_SIZE = 80;
+
+const FRAME_WIDTH = 400;
+const FRAME_HEIGHT = 100;
+const CLAMP_MARKER_HEIGHT = 40;
+
+interface ClampPlaygroundOptions {
+  lowerBound: number;
+  upperBound: number;
+}
+
+interface Props {
+  options: ClampPlaygroundOptions;
+}
+export default function App({ options }): Props {
+  const config = {
+    damping: 3,
+  };
+
+  const clampedStyle = useAnimatedStyle(() => {
+    return {
+      left: withClamp(
+        { min: options.lowerBound, max: options.upperBound },
+        withRepeat(
+          withSequence(
+            withDelay(2000, withSpring(BOX_STOP, config)),
+            withTiming(BOX_START, { duration: 0 })
+          ),
+          -1,
+          true
+        )
+      ),
+    };
+  });
```

Comment:
```suggestion
  const clampedStyle = useAnimatedStyle(() => {
    return {
      transform: [
        {
          translateX: withClamp(
            { min: options.lowerBound, max: options.upperBound },
            withRepeat(
              withSequence(
                withDelay(2000, withSpring(BOX_STOP, config)),
                withTiming(BOX_START, { duration: 0 })
              ),
              -1,
              true
            )
          ),
        },
      ],
    };
  });
```

Always prefer `transform` to position props like `left`, `right`, `top`, `bottom` as they trigger layouting process and can cause jitter (see the borders)

`left`:

https://github.com/software-mansion/react-native-reanimated/assets/39658211/815dddf8-37e8-4eb7-ab1f-26dc214eda43


`translateX`:

https://github.com/software-mansion/react-native-reanimated/assets/39658211/a8f6ffba-9f2f-4167-bc41-7ab1bb3222d8